tutorials/xr/openxr_render_models.rst

```diff
@@ -0,0 +1,266 @@
+    tracking poses to properly offset the render models.
+
+    This falls beyond the scope of this document but an example of
+    this implementation can be found in
+    `our hand tracking demo <https://github.com/godotengine/godot-demo-projects/tree/master/xr/openxr_hand_tracking_demo>`_.
+
+    **Need to submit a PR to the hand tracking demo for this implementation once render models is merged**
+
+Render models example
+~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: img/openxr_render_models_setup.webp
+
+In this setup we find an :ref:`OpenXRRenderModelManager<class_OpenXRRenderModelManager>`
+node directly underneath our :ref:`XROrigin3D<class_XROrigin3D>` node.
+On this node our ``target`` property is set to ``None set`` and will handle
+showing all render models that are currently not related to our left hand
+or right hand controllers.
+
+We then see the same setup for our left and right hand so we'll focus on
+just the left hand.
+
+We have a :ref:`XRController3D<class_XRController3D>` that will track the
+location of our hand.
+
+.. note::
+    We are using the ``grip`` pose in this example. The ``palm`` pose is
+    arguably more suitable and predictable however it is not supported
+    by all XR runtimes. See the hand tracking demo project for a
+    solution on switching between these poses based on what is supported.
+
+As a child of the node we have a :ref:`AnimatableBody3D<class_AnimatableBody3D>`
+node that follows the tracked location of the hand **but** will interact
+with physics objects to stop the players hand from going through walls etc.
+This node has a collision shape that encapsulates the hand.
+
+.. note::
+    It is important to set the physics priority such that this logic runs 
+    after any physics logic that moves the XROrigin3D node or the hand
+    will lag a frame behind.
+
+The script below shows a basic implementation for this that you can build
+upon.
+
+.. code-block:: gdscript
+
+    class_name CollisionHands3D
+    extends AnimatableBody3D
+
+    func _ready():
+        # Make sure these are set correctly
+        top_level = true
+        sync_to_physics = false
+        process_physics_priority = -90
+
+    func _physics_process(_delta):
+        # Follow our parent node around
+        var dest_transform = get_parent().global_transform
+
+        # We just apply rotation for this example
+        global_basis = dest_transform.basis
+
+        # Attempt to move to where our tracked hand is
+        move_and_collide(dest_transform.origin - global_position)
+
+
+Finally we see another :ref:`OpenXRRenderModelManager<class_OpenXRRenderModelManager>`
+node, this one with ``target`` set to the appropriate hand and
+``make_local_to_pose`` set to the correct pose.
+This will ensure that the render models related to this hand are properly
+shown and offset if our collision handler has altered the location.
+
+**Add video or animation to see this in practice**
```

Comment:
Another TODO!

Reply:
That's what I get for there being months between writing this and being able to publish it :P

Need to make a recording of my Flynn demo